src/templates/actors/partials/character-inventory-tab.html

```diff
@@ -375,7 +375,7 @@ <h4 class="item-name" title="{{item.name}}">{{item.name~}}</h4>
         </div>
         <div class="field-short">
           {{#if (eq @root.config.encumbrance
-          "itembased")}}{{ceil (mult item.system.quantity.value item.system.itemslots)}}{{else}}{{item.system.weight}}{{/if}}{{mult item.system.quantity.value item.system.weight}}
```

Comment:
We're repeating this pattern a few times. I think we should consider doing this on the datamodel.

Reply:
I could add new fields for calculated weight, slots and value to the item item.
Though with development on the new character sheet in full swing maybe we can abide a little inelegance on this one for a bit.
Is this expected to be useful in the future? Otherwise this would just add more bloat.

Reply:
An `item.system.weightCumulative` getter would be helpful. We'll need to show total weight per Item on whatever our new character sheet is, too!